pubtools/_quay/quay_client.py

```diff
@@ -40,7 +40,7 @@ def __init__(self, username, password, host=None):
         self.password = password
         self.session = QuaySession(hostname=host, api="docker")
 
-    def get_manifest(self, image, raw=False, manifest_list=False, v2s1_manifest=False):
+    def get_manifest(self, image, raw=False, manifest_list=False, media_type=None):
```

Comment:
Can you mention these changes in the commit message?

Reply:
sure